lib/beaker/host/cisco.rb

```diff
@@ -70,8 +70,12 @@ def package_config_dir
     # @return [String] Command string as needed for this host
     def prepend_commands(command = '', user_pc = '', opts = {})
       return user_pc unless command.index('vsh').nil?
+      if self[:platform] =~ /cisco_nexus/
```

Comment:
@LuvCurves I'm surprised this conditional is necessary; shouldn't this be assumed to match the `cisco` case [here](https://github.com/puppetlabs/beaker/blob/master/lib/beaker/host.rb#L60)? Do we have multiple cisco platforms, some that match `/cisco/` but don't match `/cisco_nexus/`?